end_of_run_workflow.py

```diff
@@ -14,4 +14,5 @@ def end_of_run_workflow(stop_doc):
     uid = stop_doc["run_start"]
     data_validation(uid, return_state=True)
     exporter(uid)
```

Comment:
Now that we have two exporters, let's rename this method to help the reader identify and differentiate the two exporters. Something like `xanes_exporter` should be enough.